wdl-engine/Cargo.toml

```diff
@@ -18,6 +18,7 @@ anyhow = { workspace = true }
 ordered-float = { workspace = true }
 indexmap = { workspace = true }
 serde_json = { workspace = true }
+serde_yaml = "0.9.34"
```

Comment:
As mentioned in the Sprocket PR, we should be using `serde-yaml-ng` as `serde-yaml` is unmaintained; we have a CI check in this repository that ensure we don't use unmaintained dependencies, so if we want these changes through CI it would fail.

Reply:
So the `deny` CI action is passing because apparently `serde-yaml` did not issue an "unmaintained" security advisory from what I can tell.

Still, we should no longer be using it as they still might publish a security advisory in the future.